### ISSUE TITLE
[FEATURE] Update feature for existing entries

### ISSUE THREAD

**Issue by SaintNandarOo** [Baboo7/strapi-plugin-import-export-entries#58]
**Is your feature request related to a problem? Please describe.**
Since we can't update the existing entries, duplicate entries are created when importing the same data.

**Describe the solution you'd like**
Update existing data not only with ID but also with unique field OR user-defined field.
It would also be useful if we could choose which field to update.

**Comment by Baboo7** [comment]
Hey @SaintNandarOo FYI I just published version `1.15.0` where export/import v2 has been implemented. Unique fields customisation will be way easier with the new file structure used, so this is something you will see implemented in a near future. As a disclaimer, in this v2, I will temporarily drop support of csv to fully focus on json since I have limited resources. The v1 of csv export/import will still be accessible but not maintained.

I would love to have your feedback on this new version. Feel free to join the [discord server](https://discord.gg/cGRJh7M7) of the project I just opened 😉
